src/shogun/distributions/KernelDensity.h

```diff
@@ -50,8 +50,8 @@ enum EEvaluationMode
 
 /** @brief This class implements the kernel density estimation technique. Kernel density estimation is a non-parametric
  * way to estimate an unknown pdf. The pdf at a query point given finite training samples is calculated using the
- * following formula : \\
- * \f$pdf(x')= \frac{1}{nh} \sum_{i=1}^n K(\frac{||x-x_i||}{h})\f$ \\
+ * following formula: \n
```

Comment:
I think there are centred math tex commands in doxygen, so you dont have to do newlines by hand. See e.g. GaussianKernel.h
```
 * \f[
 * k({\bf x},{\bf x'})= exp(-\frac{||{\bf x}-{\bf x'}||^2}{\tau})
 * \f]
```